README.md

```diff
@@ -1,5 +1,9 @@
 # Avalanche Security Audits
 
+## Avalanchego
+
+* [OpenZeppelin MerkleDB Audit](./avalanchego/Ava_Labs_MerkleDB_Audit.pdf)
```

Comment:
Please include a date (or at least month + year) of all audit report, like the ones for Teleporter. This helps to identify the appropriate report for review.

Reply:
addressed